katz/src/main/kotlin/katz/arrow/FunctionK.kt

```diff
@@ -16,7 +16,7 @@
 
 package katz
 
-interface FunctionK<F, G> {
+interface FunctionK<in F, out G> {
```

Comment:
AFAIK covariant is generally more flexible than invariant, right? Why wouldn't we want it?